src/zmq/models/distributions.jl

```diff
@@ -0,0 +1,8 @@
+export
+    Packed_MvNormal
+
+mutable struct Packed_MvNormal
+  mean::Vector{Float64}
+  cov::Vector{Float64}
+  distType::String
```

Comment:
Maybe clarify why `distType` is needed with a comment for users?

Reply:
This is for unpacking from JSON type, where it is  assumed the type information is available internally.